source/operators/apiOperatorApisix/dockerfile

```diff
@@ -0,0 +1,18 @@
+# Stage 1: Build stage
+FROM python:3.9 AS build
```

Comment:
In DependentAPI- and SecretsManagement-Operator we used an Python 3.12.
So, you can upgrade to v3.12 also here.

You could also switch to the alpine baseimage, which is from the security perspective more lean.
But then you also need to install gcc to work around a libcrypto bug, as you can see here:
https://github.com/tmforum-oda/oda-canvas/blob/master/source/operators/dependentApiSimpleOperator/docker/Dockerfile#L1-L8

Reply:
changed to same version v3.12 to match with DependentAPI- and SecretsManagement-Operator python version